CredentialProvider.Microsoft/CredentialProviders/Vsts/IAuthUtil.cs

```diff
@@ -70,7 +72,9 @@ public async Task<Uri> GetAadAuthorityUriAsync(Uri uri, CancellationToken cancel
             // Return the common tenant
             var aadBase = UsePpeAadUrl(uri) ? "https://login.windows-ppe.net" : "https://login.microsoftonline.com";
             logger.Verbose(string.Format(Resources.AADAuthorityNotFound, aadBase));
-            return new Uri($"{aadBase}/common");
+
+            var tenant = EnvUtil.MsalEnabled() ? OrganizationsTenant: CommonTenant;
```

Comment:
Does MSAL not support MSA users? Azure DevOps is not limited to AAD customers. In fact, given the code just above this, it's likely only MSA-backed orgs get to this part,

Reply:
Since you are using the visual studio appid which uses MSA passthrough, you cannot sign into AAD on the V2 endpoint using common. You must use organizations. 

If You were to get a different clientID which does not use MSA passthrough then you can sign in using consumers or common.

Reply:
If we got a new app id without "MSA passthrough", how would that affect MSA users? Would they still be able to sign in to Azure DevOps through this tool? How about users who still have both MSA and AAD identities with the same UPN?